src/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>CSS4 Training</title>
```

Comment:
Thanks, lets added from start. About 'user-scalable=no', lets discuss with Carmen, to let user scale, or not scale.